hooks/tk-nuke_scene_operations.py

```diff
@@ -39,10 +42,30 @@ def scan_scene(self):
         Toolkit will scan the list of items, see if any of the objects matches
         any templates and try to determine if there is a more recent version
         available. Any such versions are then displayed in the UI as out of date.
+    
         """
 
         reads = []
 
+        # If we're in Nuke Studio or Hiero, we need to see if there are any
+        # clips we need to be aware of that we might want to point to newer
+        # publishes.
+        if self.parent.studio_enabled or self.parent.hiero_enabled:
+            import hiero.core
+
+            for project in hiero.core.projects():
+                for clip in project.clipsBin().clips():
+                    files = clip.activeItem().mediaSource().fileinfos()
+                    for file in files:
+                        path = file.filename().replace("/", os.path.sep)
```

Comment:
maybe use [`ShotgunPath.normalize`](http://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.util.ShotgunPath.normalize) here?

Reply:
I'm going to stick with this to match other logic in this hook. It's old code, and I don't really want to rock the boat.